src/Parser/Hunter/Survival/Modules/Items/NesingwarysTrappingTreads.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+
+/**
+ * Nesingwary's Trapping Treads
+ * Equip: Gain 25 Focus when one of your traps is triggered.
+ */
+
+const FOCUS_PER_PROC = 25;
+
+class NesingwarysTrappingTreads extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  focusGain = 0;
+  possibleGain = 0;
+  focusWaste = 0;
+  on_initialized() {
+    this.active = this.combatants.selected.hasFeet(ITEMS.NESINGWARYS_TRAPPING_TREADS.id);
+  }
+
+  on_toPlayer_energize(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.NESINGWARYS_TRAPPING_TREADS_FOCUS_GAIN.id) {
+      return;
+    }
+    this.focusGain += event.resourceChange - event.waste;
+    this.focusWaste += event.waste;
+    this.possibleGain += FOCUS_PER_PROC;
+  }
+
+  item() {
+    return {
+      item: ITEMS.NESINGWARYS_TRAPPING_TREADS,
+      result: (
+        <dfn data-tip={`You procced Nesingwary's Trapping Treads ${this.possibleGain / FOCUS_PER_PROC} times.`}>
+          <span>You gained {this.focusGain} out of {this.possibleGain} possible focus.</span>
```

Comment:
Would be better to show a fight length independant number imo, like "Focus gained per minute". Also relegate the overcapped amount to the tooltip IMO.

Also, `<Wrapper>` pls

Reply:
I thought we didn't use `<Wrapper>` on things like this 🤔

Reply:
you don't use any extra wrapper for this, dfn already serves as one